spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+include SessionsHelper
+
+RSpec.describe UsersController, type: :controller do
+
+  before do
+    Rails.application.load_seed
+  end
+
+  let(:user) { User.find_by(username: 'user1') }
+
+  context "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+  context "GET #show" do
+    it "returns http success" do
+      get :show, params: { id: 1 }
+      expect(response).to have_http_status(:success)
+    end
+  end
+  context "GET #edit" do
```

Comment:
It fails 🔴

Reply:
We created a brand new clone to try and recreate this failing test and have not observed it failing. Is it still failing on your local machine when you run it? Or are we missing something?

Reply:
I used to have some posts with not existing user_id in both DBs: development and test 🤔 
I think it not deserves an investigation how they ended up there. They are not longer created:)